### PR TITLE
[FIX][mass_editing] Fixing error when removing value for fields being translatable,  as their translations were not removed.

### PR DESCRIPTION
**Pull request by BT-rmartin**
Module Mass Editing

Fixing error when removing value for fields being translatable,  as their translations were not removed.

### CODE REVIEW

File: mass_editing/wizard/mass_editing_wizard.py [hunk 1]
```diff
@@ -194,6 +196,18 @@ def create(self, cr, uid, vals, context=None):
...
                         dict.update({split_key: False})
+                        # If field to remove is translatable, its translations have to be removed
+                        model_field_id = model_field_obj.search(cr, uid, [('model', '=', context.get('active_model')),
+                                                                          ('name', '=', split_key)])
+                        if model_field_id and \
+                                model_field_obj.browse(cr, uid, model_field_id, context=context).translate:
+                            translation_ids = translation_obj.search(cr, uid, [
+                                                                        ('res_id', 'in', context.get('active_ids')),
+                                                                        ('type', '=', 'model'),
+                                                                        ('name', '=', "{0},{1}".format(
```
Comment (yajo) [inline mass_editing/wizard/mass_editing_wizard.py]: Must use unicode strings when calling `.format`

**Review by yajo — CHANGES_REQUESTED** [review]

**Review by yajo — APPROVED** [review]

**Comment by pedrobaeza** [comment]
Please clean your branch from undesired merge commits.

**Comment by BT-rmartin** [comment]
@pedrobaeza I already cleaned it. Thanks for the input

**Comment by pedrobaeza** [comment]
You are overpassing the 80 cols limit. Please reformat your code to fill in this space.

**Comment by BT-rmartin** [comment]
@pedrobaeza Done

**Review by pedrobaeza — APPROVED** [review]

**Pull request merged**